lib/node_modules/@stdlib/utils/compact-adjacency-matrix/test/test.js

```diff
@@ -20,31 +20,94 @@
 
 // MODULES //
 
-var tape = require( 'tape' );
-var instanceOf = require( '@stdlib/assert/instance-of' );
-var CompactAdjacencyMatrix = require( './../lib' );
+var tape = require('tape');
```

Comment:
All of these style changes should not have been made. Please revert and ensure that you follow project conventions.